src/Cookie.php

```diff
@@ -563,7 +559,10 @@ public static function fromCookieString(string $string): self
     }
 
     /**
+     * @param string $attribute
      * @psalm-return non-empty-list<null|string>
+     *
+     * @return array
```

Comment:
```suggestion
     * @psalm-return non-empty-list<null|string>
```

Reply:
phpdoc without @param and @return... why?

Reply:
because phpdocs and typehints are the same.